doc/whats_new/v0.20.rst

```diff
@@ -501,6 +525,12 @@ Support for Python 3.3 has been officially dropped.
 - |Feature| :func:`manifold.t_sne.trustworthiness` accepts metrics other than
   Euclidean. :issue:`9775` by :user:`William de Vazelhes <wdevazelhes>`.
 
+- |Fix| Fixed a bug in :func:`manifold.spectral_embedding` where the
+  normalization of the spectrum was using a division instead of a
+  multiplication. :issue:`8129` by :user:`Jan Margeta <jmargeta>`,
```

Comment:
#9062 ?

Reply:
Yeah, we're inconsistent all over the place with whether we reference PR or issue. I'm not going to be pedantic about this, especially now that I have a script (included here) which tries to break down the differences with the GitHub AI.